### PR TITLE
Assignment-4

### PR DESCRIPTION
**Pull request by abhishek-dash**

### CODE REVIEW

File: format.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            </ol></li>
+            <li>Spring <ol type="i">
+                <li>October</li>
+                <li>November</li>
+                <li>December</li>
+            </ol></li>
+        </ul>
+    </div>
+</body>
+</html>
```
Comment (G4GANCHAUDHARY) [inline format.html:68]: Did a decent job there!

File: image.html [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<a href="https://www.w3schools.com"><img src="https://www.irishtimes.com/polopoly_fs/1.4513148.1616011727!/image/image.jpg_gen/derivatives/landscape_490/image.jpg" alt="image" width="50" height="50" border=0 align='right'></a> 
+<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Iusto optio nobis exercitationem placeat animi ducimus. Aliquid est quae dolores rerum atque hic tempore quas? Laborum eum explicabo corrupti esse rem!</p>
```
Comment (G4GANCHAUDHARY) [inline image.html:2]: You could have used super tags there but this also does the job. I would suggest to always give basic super tags to html file.

**Review by G4GANCHAUDHARY — CHANGES_REQUESTED** [review]
Overall, you have completed your assignment. but, I would suggest you to go through the comment on image.html file.

**Comment by saikiransathpadi** [comment]
for activity 1, layout is correct could have used CSS to look better
try to use target="_blank" to open the page in a different tab on clicking the image
try to format the code to look the code cleaner

**Comment by Nocturnal22** [comment]
fb/twitter/insta/linkedin clickbaits not working.

**Pull request merged**